pkg/perf/healthcheck/ntp.go

```diff
@@ -57,20 +68,97 @@ func ntpCheck() error {
 	return nil
 }
 
-func getCurrentUTCTime() (time.Time, error) {
+func getCurrentUTCTime(zcl zbus.Client) (time.Time, error) {
+
+	// TimeServer represents a time server with its name and fetching function
+	type TimeServer struct {
+		Name string
+		Func func() (time.Time, error)
+	}
+
+	// List of time servers
+	var timeServers = []TimeServer{
+		{
+			Name: "worldtimeapi",
+			Func: getWorldTimeAPI,
+		},
+		{
+			Name: "worldclockapi",
+			Func: getWorldClockAPI,
+		},
+		{
+			Name: "timeapi.io",
+			Func: getTimeAPI,
+		},
+		{
+			Name: "tfchain",
+			Func: func() (time.Time, error) {
+				return getTimeChainWithZCL(zcl)
+			},
+		},
+	}
```

Comment:
Can this be defined as a global `var` (in ntp.go)

Reply:
Problem is I couldn't find a nicer way to inject zcl or otherwise I would have changed the signature of all of them to accept zbus client :(